crimsobot/bot.py

```diff
@@ -54,13 +55,42 @@ async def start(self, *args: Any, **kwargs: Any) -> None:
             flat=True
         )  # type: List[int]
         self.banned_user_ids = banned_user_ids
-
+        self.markov_cache = {
+            'crimso': m.CachedMarkov(
+                c.clib_path_join('text', 'crimso.txt'),
+                NewlineText,
+                state_size=2,
+                retain_original=False
+            ),
+            'rovin': m.CachedMarkov(
+                c.clib_path_join('text', 'rovin.txt'),
+                Text,
+                state_size=3
+            ),
+            'wisdom': m.CachedMarkov(
+                c.clib_path_join('text', 'wisdom.txt'),
+                Text,
+                state_size=3,
+            ),
+            'poem': m.CachedMarkov(
+                [c.clib_path_join('text', 'all.txt'), c.clib_path_join('text', 'randoms.txt')],
+                Text,
+                combine_weights=[1, 2],
+                state_size=2,
+                retain_original=False
+            ),
+        }
```

Comment:
not sure i'd put the source of truth for this separate from all the actual markov code?
could see something like `m.initialize_cache()`
but tbh it's not a huge issue